guizero/Waffle.py

```diff
@@ -20,9 +66,9 @@ def __init__(self, master, height=3, width=3, dim=20, pad=5, color="white", dott
         self._enabled = True
 
     	# Description of this object (for friendly error messages)
-        self.description = "[Waffle] object ("+str(height)+"x"+str(width)+")"
+        self.description = "[Waffle] object ({}x{})".format(height, width)
 
-        self.update_command(command)
+        self._command = command
         self._height = height       # How many pixels high
```

Comment:
you should be using ._update_command here which you seem to have removed? perhaps your branch was out of date?